src/pushcollector/_impl/collector.py

```diff
@@ -10,12 +10,34 @@ class Collector(object):
     Instances of this class shouldn't be constructed directly; rather,
     the :meth:`~pushcollector.Collector.get` method should be used to
     obtain a collector of the desired backend.
+
+    Custom backends *may* choose to implement the context manager protocol
+    (i.e. ``__enter__``, and ``__exit__``). If a backend does support the
+    context manager protocol, the ``__enter__`` and ``__exit__`` methods
+    may be called more than once. Whether a backend is single-use,
+    reentrant, or reusable is an implementation detail left to the
+    programmer, and no attempt is made to handle any exceptions that may
+    result from calling a backend's ``__enter__`` method more than once.
+
+    .. versionadded:: 1.3.0
+        Backends can choose to support the context manager protocol.
+
+    See Also:
```

Comment:
I suppose this should use the proper sphinx directive, https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-seealso 

```suggestion
    .. seealso::
```